### PR TITLE
homepage, navbar, style  added

### PR DESCRIPTION
**Pull request by shraddha15-creator**
Added Homepage, Navbar & styling

homepage

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Desing is amazing. Go ahead !!

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
-			<Link to="/timer">Timer</Link>
-			<Link to="/settings">Settings</Link>
+		<div className="pomo-navbar-container">
+			<div className="brand-details">
+				<Link to="/">
+					<span className="brand-name">pomoDone!</span>
+				</Link>
+			</div>
+			<div className="navbar-links">
+				<Link to="/tasks">
```
Comment (muskaanshah) [inline src/components/Navbar/Navbar.js:14]: Maybe you can change this to NavLink so that you can accommodate active classes as well

**Review by muskaanshah — APPROVED** [review]
Ui looks good

File: src/App.css [hunk 2]
```diff
@@ -1 +1,20 @@
-@import url("https://unfold-ui.netlify.app/css/components.css");
\ No newline at end of file
+@import url("https://unfold-ui.netlify.app/css/components.css");
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
+
```
Comment (JowelTisso) [inline src/App.css:3]: You can do the import in index.html

**Review by JowelTisso — APPROVED** [review]
* Check your code indentation or use prettier
* Good small PR
* Looks good to merge

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (shraddha15-creator) [reply src/components/Navbar/Navbar.js:14]: sure will do that. Thank you for review!

File: src/App.css (hunk 2 above)
Comment (shraddha15-creator) [reply src/App.css:3]: Thank you for pointing it out, will do that

**Pull request merged**